packages/create-cloudflare/README.md

```diff
@@ -37,7 +37,7 @@ Currently supported framework options: `angular`, `astro`, `docusaurus`, `gatsby
 To create a new Javascript "Hello World" worker, run:
 
 ```bash
-npm create cloudflare@latest hello-world -- --type simple --ts false
+npm create cloudflare@latest hello-world -- --type simple --no-ts
```

Comment:
This still has the extra `--` 🙂

Reply:
@Skye-31 `npm` actually requires the first set so it'll accept those args. The difference before was I had two sets of `--` like this: 

`npm create cloudflare@latest -- --type webFramework -- --framework <frameworkName>`

Reply:
Ah I misread the initial one then 😅